index.html

```diff
@@ -2,9 +2,134 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>My Journey with ACA</title>
+    <link type="text/css" rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
   </head>
-  <body>
-    <h1>Hello world!</h1>
-  </body>
+<body>
+  <header> <br><br>
+      <h1>Heather<strong>Barr</strong></h1>
+      <h3>ACA: 101 Intro to Web</h3>
+  </header>
+  <main>
+    <!-- these are my blocks with brief summary on project and a link included -->
+    <section class="week">
+        <h2>Week 01</h2>
+        <h4>Resume</h4>
+        <p>This was my first homeowrk assignment with ACA! During week one, we went over html and were to create a resume using html. I love CSS, so I had to use some.<br></p>
+        <p><em>What I used;</em></p>
+            <li>Basic HTML</li>
+            <li>Basic CSS</li>
+            <li>Column Use</li><br>
+        <button type="button"><a href="https://rawgit.com/hnbarr/web-workbook/resume/01week/resume/index.html">Check it Out!</a></button>
```

Comment:
The link is incorrect here. You are linking it to the "live" page. All you have to do is find it in your web-workbook and like it that way. Since all of your projects are under the same folder (web-workbook)

Reply:
Hey Dinh, all my folders don't have content anymore, I've been trying to locate them on my computer and they're just gone. Since they were pushed they aren't in my web-workbook folder anymore. Is that supposed to happen?